### DISCUSSION TITLE
Black outline from clip path in IM 6 but not IM 7

### DISCUSSION THREAD

**Discussion by fmw42** [ImageMagick/ImageMagick#6226] [Digital Image Processing]
I am forwarding this from a question on Stackoverflow. I do not know if this is a bug in IM 6 or from a difference for example in alpha behavior between IM 6 and 7.

Input:

![61270542](https://user-images.githubusercontent.com/13458218/229908428-6487d292-7232-4422-8b64-6f0ec011726f.jpg)

IM 6.9.12.82

`convert -quiet 61270542.jpg -alpha transparent -clip -alpha opaque -strip 61270542_clipped1_6.jpg`

![61270542_clipped1_6](https://user-images.githubusercontent.com/13458218/229908466-d6bf27b0-c726-4c8c-92e6-f765dc30bbea.jpg)

IM 7.1.1.5

`magick -quiet 61270542.jpg -alpha transparent -clip -alpha opaque -strip 61270542_clipped1_7.jpg`

![61270542_clipped1_7](https://user-images.githubusercontent.com/13458218/229908504-a20ef2df-e2c5-4795-8eb8-9102063d085a.jpg)

**Comment by urban-warrior** [comment]
Thanks for the problem report.  We can reproduce it and will have a patch to fix it in the GIT main branch @ https://github.com/ImageMagick/ImageMagick later today.  The patch will be available in the beta releases of ImageMagick @ https://imagemagick.org/archive/beta/ by sometime tomorrow.
